docs/Design.md

```diff
@@ -43,19 +44,19 @@ blockchain and mempool do not block the master process very much. It also means
 transaction verification can be parallelized.
 
 Strangely enough, workers are faster in the browser than they are in node since
-you are allowed to share memory between threads using the transferrable api
+you are allowed to share memory between threads using the transferable API
 (Uint8Arrays can be "transferred" to another thread). In node, you have to pipe
 data to another process.
 
 But of course, there is a benefit to having a multi-process architecture: the
 worker processes can die on their own without disturbing the master process.
 
-Bcoin uses [secp256k1-node][secp256k1-node] for ecdsa verification, which is a
+Bcoin uses [secp256k1-node][secp256k1-node] for ECDSA verification, which is a
```

Comment:
@braydonf is this still true? Or didn't JJ rewrite the binding for bcrypto? Looks some things were taken: https://github.com/bcoin-org/bcrypto/blob/634cad73e872a5495e1fe7d25628d52b539fca24/utils/has_lib.sh#L3

Reply:
`bcrypto` [vendored](https://github.com/bcoin-org/bcrypto/tree/393b34a2c9b8589f7e550ffd857e56315d1ec3be/vendor/secp256k1) secp256k1-node, and recently [moved to the main module](https://github.com/bcoin-org/bcrypto/commit/634cad73e872a5495e1fe7d25628d52b539fca24).